scripts/dist-tags-promote

```diff
@@ -1,5 +1,14 @@
+# get CI environment auth'd; local one-off use assumes you're already auth'd to npm
+if [ $NPM_TOKEN ]; then
+  echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
+fi
+
 CLI_RC_VERSION=$(npm show sfdx-cli --json | jq -r '.["dist-tags"] | .["latest-rc"]')
+# use the following line for testing without making changes
+# npm dist-tag add sfdx-cli@${CLI_RC_VERSION} latest-rc
```

Comment:
What are these lines for? Are they really necessary?

Reply:
easy way to replicate the "dry run" feature we had before.  Removing.